### ISSUE TITLE
support for filtering errors?

### ISSUE THREAD

**Issue by simon-weber** [getsentry/raven-python#851]
I'm looking for similar functionality to https://github.com/getsentry/raven-js/issues/435#issuecomment-183021031, but for python. Is there some way to do this? I found the processors api, but [they seem to only allow of events rather than filtering of them](https://github.com/getsentry/raven-python/blob/f468822feba6081524ca6783f9489c8ba4dc8d9c/raven/base.py#L451).

**Comment by mitsuhiko** [comment]
Currently you can override the `should_capture` method of the client to achieve what you are after but there should probably be a better api.

**Comment by tp7** [comment]
I have the same problem, in particular I want to rate limit exceptions on the client to e.g. 1 exception with particular fingerprint every minute.

Is `should_capture` still the recommended way to do something like this? Seeing how it doesn't have a doc string I wonder if it's actually considered a part of the public API and is safe to use in the long run.

**Comment by dcramer** [comment]
Yep should_capture is the best method. Even if we changed it we would make it very easy to upgrade.  That said no plans to tweak it and you can consider it public.
